### PR TITLE
Improve `rate_up/down` tooltips, pitch vs. speed

### PR DESCRIPTION
**Pull request by glocq**
The tooltip help regarding the playback rate change buttons stated that those raise/lower pitch.

If keylock is disabled, this is true, though incomplete, since the buttons also affect playback speed.
If keylock is enabled, this is false, since pitch is left unchanged, and only playback speed changes.

This change replaces the tooltip text about pitch change with text about playback speed. It also copies the clarification:
"(affects both the tempo and the pitch). If keylock is enabled, only the tempo is affected."
over from the Speed Control fader tooltip, to prevent further confusion.

### CODE REVIEW

**Comment by JoergAtGithub** [comment]
Welcome at Mixxx!
As a first-time contributor we need you to sign the [Mixxx Contributor Agreement](https://docs.google.com/forms/d/e/1FAIpQLScC9QG327sjLL0eWftmfGUasxFFLxg6LCXJ2xHDYRzFIRqyiw/viewform?formkey=dEpYN2NkVEFnWWQzbkFfM0ZYYUZ5X2c6MQ) and comment here when you have done so. It gives us permission to distribute your contribution under the GPL v2 or later license and the Apple Mac App Store. It is also helpful for us to have contact information for contributors in case we may need it in the future.

**Comment by glocq** [comment]
Hi, I have signed the contributor agreement.

**Comment by daschuer** [comment]
I can unfortunately not confirm your signing. Something has probably failed. Can you please repeat the signing? 
Sorry for the hassle.

**Comment by glocq** [comment]
Sorry, I just tried again, hope it ended up working?

**Comment by daschuer** [comment]
Thank you. I can confirm your signature.

**Comment by ronso0** [comment]
@glocq Please take a look at the failing `pre-commit` hook. The linebreaks ned to be adjusted.
See https://github.com/mixxxdj/mixxx/blob/main/CONTRIBUTING.md#git-workflow -> pre-commit

**Comment by glocq** [comment]
Thank you for the heads up, sorry I didn't get the chance to update my code yet. Will do so in the next days.

File: src/skin/legacy/tooltips.cpp
Comment (ronso0) [inline src/skin/legacy/tooltips.cpp:329]:
```suggestion
            << QString("%1: %2").arg(leftClick, tr("Raises the track playback speed (tempo). If keylock is disabled, pitch is also affected."))
```

File: src/skin/legacy/tooltips.cpp
Comment (ronso0) [inline src/skin/legacy/tooltips.cpp:335]:
```suggestion
            << QString("%1: %2").arg(leftClick, tr("Lowers the track playback speed (tempo). If keylock is disabled, pitch is also affected."))
```

File: src/skin/legacy/tooltips.cpp
Comment (ronso0) [inline src/skin/legacy/tooltips.cpp:341]:
```suggestion
            << QString("%1: %2").arg(leftClick, tr("Holds playback speed higher while active (tempo). If keylock is disabled, pitch is also affected."))
```

File: src/skin/legacy/tooltips.cpp
Comment (ronso0) [inline src/skin/legacy/tooltips.cpp:347]:
```suggestion
            << QString("%1: %2").arg(leftClick, tr("Holds playback speed lower while active (tempo). If keylock is disabled, pitch is also affected."))
```

Comment (ronso0) [reply src/skin/legacy/tooltips.cpp:347]: Just noticed the redundancy.
In order to minimize translation effert, let's use a const QString for "If keylock is disabled, pitch is also affected." and append that. See `QString changeAmount` above for example.

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Comment by glocq** [comment]
Hi @ronso0 , sorry I completely forgot about this pull request. I accepted the changes you did implement, however I still think I have some style changes to make. I am trying to install pre-commit right now. Regarding your suggestion to make a common QString, that sounds good, I am not sure I would be the best person to implement it though.

**Comment by glocq** [comment]
An update: I did run pre-commit, it says the check "don't commit to branch" failed. I am not sure what this even means :/ Obviously there are commits to my branch?

**Comment by ronso0** [comment]
what does `git status` print?

**Comment by glocq** [comment]
Ah sorry, didn't read your answer about breaking lines before talking about my not understanding the output of pre-commit.
`git status` says everything is clean and up to date, but I will try breaking lines as pre-commit suggests, that might solve the issue.

**Comment by glocq** [comment]
I had indeed missed the example QString. I implemented the change you suggested, ran pre-commit and it did not complain. Sorry for this pull request being so long-winded for such a small change. I hope the changes are adequate now.

**Comment by ronso0** [comment]
> `git status` says everything is clean and up to date

The exact output might have clues why `no-commit-to-branch` failed. IIUC that only kicks in if you want to commit/push to a branch that's named exactly like the Mixxx upstream branch, e.g. 2.4 or main.

Anyway, issue resolved I guess : )

**Comment by ronso0** [comment]
> Sorry for this pull request being so long-winded for such a small change. 

Oh, don't worry, at all, we had much longer PRs for much less changed lines...

**Comment by ronso0** [comment]
Changes LGTM, just waiting for CI

**Comment by glocq** [comment]
Thank you for your help!

**Review by ronso0 — APPROVED** [review]

**Pull request merged**
